src/nebula/NebulaAction.cpp

```diff
@@ -213,10 +227,10 @@ ResultCode WriteCircleAction::doRun() {
                                         << row;
             batchCmds.clear();
         }
-        batchCmds.emplace_back(folly::stringPrintf("%ld:(%ld)", row, row + 1));
+        batchCmds.emplace_back(folly::stringPrintf("%lu:(\"%s\")", startId_++, genData().c_str()));
```

Comment:
Is it OK for current plans in production?

Reply:
Yes，it's ok.